CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Documenting All Changes to the Skelebot Project
 ## v1.6.3
 #### Changed
 - **CI/CD** | Update PyPI workflow to trigger on prereleases only.
+- **Dependency management** | Added requirements.txt as single source of truth for dependencies.
```

Comment:
Instead of placing this in v1.6.3 we should bump to v1.6.4 and create a new section in this Changelog for 1.6.4.

Reply:
Done! (https://github.com/carsdotcom/skelebot/pull/89/commits/434f4a8993e84ade0e677342ad5d9319738c3c60)